package.json

```diff
@@ -29,6 +29,7 @@
         "gatsby-plugin-posthog-analytics": "^0.0.1",
         "gatsby-plugin-react-helmet": "^3.1.7",
         "gatsby-plugin-remove-trailing-slashes": "^2.3.11",
+        "gatsby-plugin-sass": "^2.3.16",
```

Comment:
I guess this is fine, though nowadays instead of "sass" people tend to use "postcss" with optionally some "scss" plugins. Not sure if anything to change here.

Reply:
![fine](https://user-images.githubusercontent.com/38760734/95991069-38d49700-0e24-11eb-9836-da59e4fa118f.jpeg)